app/controllers/wikis_controller.rb

```diff
@@ -0,0 +1,61 @@
+require "custom_logger"
+
+class WikisController < ApplicationController
+  def index
+    @wikis = Wiki.all
+  end
+
+  def create
+    #Rails.logger.info "We're starting the controller #{current_user.inspect}"
+    # @wiki = Wiki.new
+    # @wiki.title = params[:wiki][:title]
+    # @wiki.body = params[:wiki][:body]
+    # @wiki.private = params[:wiki][:private]
+    # @wiki.user = current_user
+    #Rails.logger.warn(params)
+    @wiki = Wiki.new(params.require(:wiki).permit(:title, :body, :private))
```

Comment:
Making a note here - no need to make changes in this branch because it's already been merged, but a note for the branch you're working now. If you've already made the change, ignore this:

You'll want to refactor `params.require(:wiki).permit(:title, :body, :private)` into a `wiki_params` method because it's 1) less typing and b) saves you from errors if one gets updated and the one on line 41 doesn't